src/drawer.js

```diff
@@ -247,13 +247,15 @@ $.Drawer.prototype = {
             var viewportSize = this._calculateCanvasSize();
             if( this.canvas.width != viewportSize.x ||
                 this.canvas.height != viewportSize.y ) {
+                var imageSmoothingEnabled = this.canvas.getContext('2d').imageSmoothingEnabled;
```

Comment:
I think what I would do would be to keep our own `_imageSmoothingEnabled` property that gets set in `setImageSmoothingEnabled` and then used here. That way you are relying on an explicit variable and not something in the Canvas that might get changed out of our control.

Reply:
I can't see this property...
Did you mean something like:
                var imageSmoothingEnabled = this.context.imageSmoothingEnabled;
?